front-end/src/components/filters/games-filter.tsx

```diff
@@ -33,43 +35,20 @@ export default function GamesFilter(props: Props) {
   return (
     <RisenBox1 title="Filters" sx={props.sx}>
       {
-        props.useSeason ? <FormControl sx={{minWidth: '300px', pt: 1, pb: 1}}>
-          <InputLabel id="season-filter-select-label">Season</InputLabel>
-          <Select
-            labelId="season-filter-select-label"
-            id="season-simple-select"
-            value={props.seasonConfig?.seasonId}
-            label="Season"
-            onChange={(event: SelectChangeEvent) => {props.seasonConfig?.setSeasonId(event.target.value)}}
-          >
-            {props.hideAllGames === true ? null : <MenuItem value="ALL">All Tournament Games</MenuItem>}
-            <MenuItem value="RISEN">All Risen Games</MenuItem>
-            {
-              props.seasonConfig?.seasons?.map(season => (
-                <MenuItem value={season.id}>{season.seasonName}</MenuItem>
-              ))
-            }
-          </Select>
-        </FormControl> : null
+        props.useSeason && <RisenSeasonSelector
+            callBack={(event: SelectChangeEvent) => {props.seasonConfig?.setSeasonId(event.target.value)}}
+            sx={{minWidth: '300px', pt: 1, pb: 1}}
+            seasonConfig={props.seasonConfig}
+            hideAllGames={props.hideAllGames !== undefined}/>
```

Comment:
This probably should be !!props.hideAllGames. Cuz passing in false evaluates to true.

Reply:
Good catch lol